.github/workflows/cypress.yml

```diff
@@ -57,13 +57,22 @@ jobs:
     - name: Test
       run: npm run cypress:run
 
-    - name: Upload artifacts
-      uses: actions/upload-artifact@v2
+    - name: Update summary
+      if: always()
```

Comment:
I pushed this in 424debc32fa93aa5623daa4f78b5f1982d7a294a to ensure the summary is displayed for both successful and unsuccessful runs.

I reversed some assertions to trigger an [unsuccessful run](https://github.com/10up/safe-svg/actions/runs/4920736514). This has since been removed from the history to remove noise.